vllm/v1/metrics/loggers.py

```diff
@@ -139,201 +145,276 @@ def log_engine_initialized(self):
             self.vllm_config.cache_config.num_gpu_blocks)
 
 
-class PrometheusStatLogger(StatLoggerBase):
+class Metrics:
```

Comment:
Given that all cls attributes are Prometheus classes, I'd suggest name this class "PrometheusMetrics".